tree/dataframe/src/RDFColumnRegister.cxx

```diff
@@ -289,6 +290,8 @@ std::string RColumnRegister::ResolveAlias(std::string_view alias) const
    return aliasStr; // not an alias, i.e. already resolved
 }
 
+/// Return a RDefineReader or a RVariationReader, or nullptr if not available.
+/// If requestedType does not match the actual type of the Define or Variation, an exception is thrown.
 RDFDetail::RColumnReaderBase *RColumnRegister::GetReader(unsigned int slot, const std::string &colName,
```

Comment:
Why doesn't this method also change to return a reference?

Reply:
this one is the outside-facing method and it actually needs to return a nullptr to signal to the outside that there is no variation or define for the requested column (otherwise we'd have to require that the "outside" checks this themselves...with a separate call to `RColumnRegister` that performs the exact same look-ups as this one).